openedx/adg/admin_dashboard/webinars/__init__.py

```diff
@@ -1,3 +1,17 @@
 """
-App for managing webinars in admin dashboard
+This app adds the ability to manage webinars in admin dashboard.
+
+ - An admin can create a webinar and then save it.
+ - Admin needs to fill information required for a webinar and can invite participants like cohosts, panelists, invitees
+ and can also invite all the users of platform or a business unit. A business unit can invite all users or users
```

Comment:
In case a particular business unit is selected, all the users belonging to that business unit are invited.